cylc/flow/simulation.py

```diff
@@ -166,39 +213,44 @@ def sim_time_check(
     Returns:
         True if _any_ simulated task state has changed.
     """
-    sim_task_state_changed = False
     now = time()
+    sim_task_state_changed: bool = False
+
     for itask in itasks:
-        if itask.state.status != TASK_STATUS_RUNNING:
+        if (
+            itask.state.status != TASK_STATUS_RUNNING
+            or itask.state.is_queued
+            or itask.state.is_held
+            or itask.state.is_runahead
```

Comment:
I don't think this is right? Why the change?

Reply:
It's not, I've removed it. 👍🏼

Reply:
Still not pushed?